examples/direct_fidelity_estimation_test.py

```diff
@@ -59,11 +58,13 @@ def test_direct_fidelity_estimation_with_noise_non_clifford():
     assert estimated_fidelity >= -1.0 and estimated_fidelity <= 1.0
 
 
+@cirq.testing.skip_if_module_not_exists(module="cirq.google")
 def test_incorrect_sampler_raises_exception():
+    import cirq.google
+
     qubits = cirq.LineQubit.range(1)
     circuit = cirq.Circuit(cirq.X(qubits[0]))
-
-    sampler_incorrect_type = cg.QuantumEngineSampler(
+    sampler_incorrect_type = cirq.google.QuantumEngineSampler(
```

Comment:
It should be possible to use `cirq.ZerosSampler` here - the method only cares if the sampler is not a `DensityMatrixSimulator`.

Reply:
good catch, thanks!